src/components/Sum.scss

```
@@ -0,0 +1,11 @@
+.addition {
```

```suggestion
.sum {
```
It's a good practice to make the name of the main class of the component should be the same as the filename